livingatlas/pipelines/pom.xml

```diff
@@ -603,6 +603,11 @@
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.jcabi</groupId>
+      <artifactId>jcabi-log</artifactId>
+      <version>0.18.1</version>
```

Comment:
Nit: suggest keep versions in properties

Reply:
I've fixed this post-merge.